romancal/scripts/make_regtestdata.sh

```diff
@@ -63,11 +80,11 @@ cp r0000101001001001001_0002_wfi10_cal.asdf $outdir/roman-pipeline/dev/WFI/image
 
 # resample regtest; needs r0000101001001001001_000{1,2}_wfi01_cal.asdf
 # builds the appropriate asn file and calls strun with it
-echo "Creating regtest files for resample..."
-asn_from_list r0000101001001001001_0001_wfi01_cal.asdf r0000101001001001001_0002_wfi01_cal.asdf r0000101001001001001_0003_wfi01_cal.asdf -o L3_mosaic_asn.json --product-name mosaic
-strun romancal.step.ResampleStep L3_mosaic_asn.json --rotation=0 --output_file=mosaic.asdf
-cp L3_mosaic_asn.json $outdir/roman-pipeline/dev/WFI/image/
-cp mosaic_resamplestep.asdf $outdir/roman-pipeline/dev/truth/WFI/image/
+#echo "Creating regtest files for resample..."
+#asn_from_list r0000101001001001001_0001_wfi01_cal.asdf r0000101001001001001_0002_wfi01_cal.asdf r0000101001001001001_0003_wfi01_cal.asdf -o L3_mosaic_asn.json --product-name mosaic
```

Comment:
`L3_mosaic_asn.json` is generated twice in this file on main (and each time the association is different). On artifactory the current `L3_mosaic_asn.json` has a target (`r274dp63x31y81`) which better matches the other place where `L3_mosaic_asn.json` is generated so this portion of the script was commented out (let me know if I should just delete these lines) and the other updated to generate the required files using a single `L3_mosaic_asn.json`.

Reply:
Go ahead and delete these lines.

Reply:
Thanks! Removed in [6050049](https://github.com/spacetelescope/romancal/pull/1533/commits/6050049d368843dacbe5a5e3827011d80b7dc232)